spec.bs

```diff
@@ -967,6 +982,64 @@ To <dfn>generate and score bids</dfn> given an [=auction config=] |auctionConfig
 
 </div>
 
+<div algorithm>
+To <dfn>build an interest group passed to generateBid</dfn> given an [=interest group=] |ig|:
+  
+  1. Let |igGenerateBid| be a new {{GenerateBidInterestGroup}} with the following fields:
+    <dl>
+      <dt>{{GenerateBidInterestGroup/owner}}
+      <dd>The [=serialization of an origin|serialization=] of |ig|'s [=interest group/owner=]
+      <dt>{{GenerateBidInterestGroup/name}}
+      <dd>|ig|'s [=interest group/name=]
+      <dt>{{GenerateBidInterestGroup/enableBiddingSignalsPrioritization}}
+      <dd>|ig|'s [=interest group/enable bidding signals prioritization=]
+      <dt>{{GenerateBidInterestGroup/priorityVector}}
+      <dd>|ig|'s [=interest group/priority vector=] if not null, otherwise {{undefined}}
+      <dt>{{GenerateBidInterestGroup/executionMode}}
+      <dd>|ig|'s [=interest group/execution mode=]
+      <dt>{{GenerateBidInterestGroup/biddingLogicURL}}
+      <dd>The [=serialize a URL|serialization=] of |ig|'s [=interest group/bidding url=]

```

Comment:
I'd try to be a little clearer in the text that this isn't just the serialization defined by the URL spec. Maybe:

```suggestion
      <dd>The [=serialize a URL|serialization-or-undefined=] of |ig|'s [=interest group/bidding url=]
```

And similarly below.